### PR TITLE
Add support for minimums in AwardLevelProportional*XP

### PR DESCRIPTION
**Pull request by LtRipley36706**

### CODE REVIEW

File: Source/ACE.Server/WorldObjects/Managers/EmoteManager.cs [hunk 1]
```diff
@@ -116,16 +116,21 @@ public float ExecuteEmote(BiotaPropertiesEmote emoteSet, BiotaPropertiesEmoteAct
 
                 case EmoteType.AwardLevelProportionalSkillXP:
 
+                    var max = emote.Max64 ?? emote.Max ?? 0;
```
Comment (gmriggs) [inline Source/ACE.Server/WorldObjects/Managers/EmoteManager.cs]: can you put min before max for all of these? (definition, param order)

Comment (LtRipley36706) [reply Source/ACE.Server/WorldObjects/Managers/EmoteManager.cs]: k

File: Source/ACE.Server/WorldObjects/Managers/EmoteManager.cs [hunk 2]
```diff
@@ -116,16 +116,21 @@ public float ExecuteEmote(BiotaPropertiesEmote emoteSet, BiotaPropertiesEmoteAct
...
 
                 case EmoteType.AwardLevelProportionalXP:
 
                     bool shareXP = emote.Display ?? false;
+                    min = emote.Min64 ?? emote.Min ?? 0;
+                    max = emote.Max64 ?? emote.Max ?? 0;
 
                     if (player != null)
-                        player.GrantLevelProportionalXp(emote.Percent ?? 0, emote.Max64 ?? 0, shareXP);
+                        player.GrantLevelProportionalXp(emote.Percent ?? 0, min, min, shareXP);
```
Comment (gmriggs) [inline Source/ACE.Server/WorldObjects/Managers/EmoteManager.cs]: min, min

Comment (LtRipley36706) [reply Source/ACE.Server/WorldObjects/Managers/EmoteManager.cs]: fixed already :)

**Pull request merged**